src/main/java/spireMapOverhaul/zones/brokenSpace/patches/BossRelicPricePatch.java

```diff
@@ -0,0 +1,25 @@
+package spireMapOverhaul.zones.brokenSpace.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+
+import java.util.Objects;
+
+@SpirePatch(
+        clz = AbstractRelic.class,
+        method = "getPrice"
+)
+public class BossRelicPricePatch {
+
+
+    @SpirePrefixPatch
+    public static SpireReturn<Integer> Prefix(AbstractRelic relic) {
+        // 14
+        if (Objects.requireNonNull(relic.tier) == AbstractRelic.RelicTier.BOSS) {
```

Comment:
This might crash for circlets or wacky modded relics, also is this even still needed? I think you removed this?